### ISSUE TITLE
Blog to evangelize asking that modules owners to test against LTS releases and Current.

### ISSUE THREAD

**Issue by mhdawson** [nodejs/package-maintenance#204]

**Comment by ghinks** [comment]
from the work I have conducted myself on the three options that seem to be available
- renovate
- greenkeeper
- dependabot

I only think that renovate met with the criterion we were discussing at the time.

Just as an individual I experimented with renovate and found that the basic setup with 

```
{
  "extends": [
    "config:base"
  ],
  "travis": {
    "enabled": true,
    "supportPolicy": ["lts_latest", "current"]
  },
  "schedule": "before 2am"
}
```

lts_latest and current worked as described.

( [here](https://github.com/ghinks/epl-ml-mono-repo/pulls?q=is%3Apr+is%3Aclosed) ) are some of the generated pulls including node version updates from renovate. 

I hope this helps.
